docs/documentation/pages/administrator/020-prerequisites.md

```diff
@@ -39,6 +39,16 @@ for avoiding confusing situations as outlined
 In the end, you won't need to create NGINX ingresses yourself: the Astarte Operator itself will take
 care of this task.
 
+
+## External RabbitMQ
```

Comment:
```suggestion
## RabbitMQ
```
Having an external rabbitmq is not a requirement. We both support in cluster and saas solutions.